search/includes/classes/class-search.php

```diff
@@ -2022,4 +2024,9 @@ public function maybe_change_index_version() {
 			$this->versioning->set_current_version_number( $indexable, $normalized_version );
 		}
 	}
+
+	public function exclude_es_query_reserved_names( $taxonomies ) {
+		global $wp;
+		return array_merge( $taxonomies, $wp->public_query_vars, $wp->private_query_vars );
```

Comment:
With the latest change, I'm seeing certain names previously on the list left off now, e.g. `author__in`. Was that the intent, @pschoffer? Or were we supposed to include all parameters in https://developer.wordpress.org/reference/classes/wp_query/#parameters?

Reply:
I am not sure to be honest... I couldn't find too much information in the original PR. I'll reach out to folks on slack to see if they remember more.